scripts/check_frontend_coverage.py

```diff
@@ -259,6 +260,7 @@
     'NumberWithUnitsObjectFactory.ts',
     'numeric-input-validation.service.ts',
     'object-editor.directive.ts',
+    'oppia-angular-root.component.ts',
```

Comment:
We don't expect to add new files in this list, can we add test for these new directives/pipes?

Reply:
This was required to unblock contributors who were facing errors with services. I have raised an issue (#9521) for this and I am writing the tests. (Will make a different pr for that).